examples/fillform.js

```diff
@@ -39,7 +39,7 @@ export default async function() {
       page.locator('input[type="submit"]').click(),
     ]);
     check(page, {
-      'header': page.locator('h2').textContent() == 'Welcome, admin!',
+      'header': page => page.locator('h2').textContent() == 'Welcome, admin!',
```

Comment:
WDYT about saying just `p` to distinguish from the actual `page` object?

```suggestion
      'header': p => p.locator('h2').textContent() == 'Welcome, admin!',
```

Reply:
Nice idea, i've resoled this in 84ca43cc7eab12d88c3c17f9ed295ebea4fba51b. I've also changed it from `'header'` to `header` 👍